projects/sandbox/src/app/doc-arrows/doc-arrows.component.ts

```diff
@@ -0,0 +1,275 @@
+import { Component, inject, Renderer2, RendererFactory2 } from '@angular/core';
+import { CommonModule, Location } from '@angular/common';
+
+export type ArrowType = 'top' | 'right' | 'bottom' | 'left';
+
+@Component({
+  imports: [CommonModule],
+  selector: 'sb-doc-arrows',
+  standalone: true,
+  templateUrl: './doc-arrows.component.html',
+  styleUrls: ['./doc-arrows.component.scss']
+})
+export class DocArrowsComponent {
+  private readonly location = inject(Location);
+  private readonly rendererFactory = inject(RendererFactory2);
+  renderer: Renderer2;
+
+  constructor() {
+    this.renderer = this.rendererFactory.createRenderer(null, null);
+    this.location.go('/');
+  }
+
+  public documentationArrows = [1];
+
+  sideIndent = '350px';
+  topIndent = '50px';
+  bottomIndent = '300px';
+
+  //  arrowDefaults: { [key: ArrowType]: CSSStyleDeclaration } = {
+  arrowDefaults = {
+    top: {
+      bottom: 'auto',
+      left: '50%',
+      right: 'auto',
+      top: this.topIndent,
+      height: '5em',
+      width: '2.5em'
+    },
+    right: {
+      bottom: 'auto',
+      left: 'auto',
+      right: this.sideIndent,
+      top: '50%',
+      height: '2.5em',
+      width: '5em'
+    },
+    bottom: {
+      bottom: this.bottomIndent,
+      left: '50%',
+      right: 'auto',
+      top: 'auto',
+      height: '5em',
+      width: '2.5em'
+    },
+    left: {
+      bottom: 'auto',
+      left: this.sideIndent,
+      right: 'auto',
+      top: '50%',
+      height: '2.5em',
+      width: '5em'
+    }
+  };
+
+  rotateArrow(arrow: HTMLElement): void {
+    const arrowTypes: Array<ArrowType> = ['top', 'right', 'bottom', 'left'];
+
+    arrowTypes.every((arrowType: ArrowType, index: number) => {
+      if (arrow.classList.contains(arrowType)) {
+        const newArrowType = arrowTypes[index + 1 >= arrowTypes.length ? 0 : index + 1];
+        const defaults = this.arrowDefaults[newArrowType];
+
+        this.renderer.removeClass(arrow, arrowType);
+        this.renderer.addClass(arrow, newArrowType);
+
+        this.renderer.setStyle(arrow, 'bottom', defaults.bottom);
+        this.renderer.setStyle(arrow, 'top', defaults.top);
+        this.renderer.setStyle(arrow, 'left', defaults.left);
+        this.renderer.setStyle(arrow, 'right', defaults.right);
+        this.renderer.setStyle(arrow, 'width', defaults.width);
+        this.renderer.setStyle(arrow, 'height', defaults.height);
+
+        return false;
+      }
+      return true;
+    });
+  }
+
+  arrowActiveKey(event: KeyboardEvent): void {
+    const tgt = event.target as HTMLElement;
+
+    if (tgt) {
+      const arrow = tgt.closest('.arrow') as HTMLElement;
+
+      if (arrow) {
+        // rotation
+        if (['r', 'R'].includes(event.key)) {
```

Comment:
only writing `r` works caps `R` at the moment is not possible to use.

![afbeelding](https://github.com/europeana/metis-ui/assets/19560266/dd3fd1cf-57c8-48e7-9aec-1de5165869a7)

Reply:
If there's a diagram using letters up to 'R' then it probably has too many arrows!

But yes, they'd have to use cap lock in this case.